best-practices.md

```diff
@@ -157,6 +157,15 @@ unique, so may need a prefix. But the use of URI or file path reserved character
 result in [percented encoded](https://tools.ietf.org/html/rfc3986#section-2) [STAC API](https://github.com/radiantearth/stac-api-spec) 
 endpoints and it prevents the use of IDs as file names as recommended in the [catalog layout](#catalog-layout) best practices.
 
+In general, we recommend that item IDs *not* include any kind of processing timestamp. If, for whatever reason, an
+item needs to be reprocessed the new item would have a different ID. This might result in "duplicate" items: multiple
+items with the same spatio-temporal footprint, which might not be desirable for end users.
```

Comment:
```suggestion
In general, we recommend that item IDs *not* include any kind of processing timestamp. If, for example, an item's assets require re-processing (e.g. to incorporate changes due to a new processing baseline), the re-processed items would have new IDs. This might result in "duplicate" items: multiple
items with the same spatio-temporal footprint, which might not be desirable for end users.
```

In the cases we've seen, it's the item's assets being reprocessed, not the item itself (i.e. the metadata). This change calls out assets as the thing being reprocessed, and tweaks the language slightly.

Reply:
Yep, that's a good distinction.